pkgs/development/web/nodejs/v23.nix

```diff
@@ -11,11 +13,10 @@ let
     inherit openssl;
     python = python3;
   };
-in
-buildNodejs {
+in buildNodejs {
   inherit enableNpm;
-  version = "23.6.1";
-  sha256 = "fefa49dede8733018ada4e30f885808cc4e22167b8ae3233c6d6a23737aff76f";
+  version = "23.7.0";
+  sha256 = "8de192ef2fee2ee8a230dd8d0e9aee182ee9c9856ccdb5fd95188abe84f77242";
```

Comment:
We should switch that over to hash and sri hashes

Reply:
No, IMO we should keep using the hashes that are signed bv Node.js releasers (https://nodejs.org/en/blog/release/v23.7.0#shasums) unless we have a good reason to switch to something less verifiable.